tests/testvectors/invalidPayloadCases.json

```diff
@@ -402,5 +402,103 @@
+  },
+  {
```

Comment:
Are these additions meant to be included in this PR?

Reply:
These files are generated by generate-transaction-tests. Maybe they should not be in the repository at all, but I see some point in having them here. This change indeed belongs to a previous PR, but as it is a generated file I did not notice that the update of the generated file was missing. 
There are several solutions to this issue
1. We may remove the generated files from the repository. This would require some changes in the Makefiles and solve the issue in this way in the next PR. 
2. We may create a separate full request for this change (although, it is possible that other generated files will change too, as many of them are generated asynchronously). 
3. We may consider this insignificant and keep the changes here.
My opinion is both 1 and 3.